stan/math/rev/mat/fun/cholesky_decompose.hpp

```diff
@@ -11,6 +11,16 @@
 #include <stan/math/prim/mat/err/check_pos_definite.hpp>
 #include <stan/math/prim/mat/err/check_square.hpp>
 #include <stan/math/prim/mat/err/check_symmetric.hpp>
+#include <stan/math/gpu/cholesky_decompose.hpp>
+#include <stan/math/gpu/constants.hpp>
+#include <stan/math/gpu/diagonal_multiply.hpp>
+#include <stan/math/gpu/copy.hpp>
+#include <stan/math/gpu/lower_tri_inverse.hpp>
+#include <stan/math/gpu/matrix_gpu.hpp>
+#include <stan/math/gpu/multiply.hpp>
+#include <stan/math/gpu/err/check_square.hpp>
+#include <stan/math/gpu/err/check_symmetric.hpp>
```

Comment:
TODO(me): Remove check_ from gpu since we don't use those here

Reply:
Sorry, for this comment I mean we use the cpu versions here, not that we don't test for these.

Though we could use the gpu version. I can play with that